pkg/cli/color-console_windows.go

```diff
@@ -0,0 +1,36 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+// +build windows
+
+package cli
+
+import (
+	"golang.org/x/sys/windows"
+	"os"
+)
+
+// In order for colored output to work on Windows, we need to explicitly
+// enable the virtual terminal processing console mode.  This is needed for
```

Comment:
Hey Jon,

One suggestion--can we detect if virtual terminal processing console mode is enabled, and either switch to no-color automatically, issue a warning, or both?

Reply:
Hi Willie, the PreRun func for scanCmd in cli/scan.go will disable colored output mode if the terminal is not a tty.

This code just ensures that Windows users will be able to see colored output.  The default mode is disabled, probably for historical reasons that are no longer relevant.

Note that Terrascan uses third-party packages like zap that also expect a color-capable terminal.  If we wanted to disable colors then we would need to figure out how/if those could be forced to a non-color mode.